composer.json

```diff
@@ -12,10 +15,14 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=5.6",
+        "equip/command": "^2.0",
         "league/event": "^2.1",
-        "psr/log": "^1.0"
+        "psr/log": "^1.0",
+        "rdlowrey/auryn": "^1.4"
```

Comment:
Since we only have `AurynCommandFactory` and no `CommandFactoryInterface`, I figured it's best to add it.